test/CheckedCRewriter/function_tests.c

```diff
@@ -166,7 +166,7 @@ int *ok_mut_clone(int *a, int b) {
   *a = b;
   return a;
 }
-//CHECK: int *ok_mut_clone(int *a, int b) {
+//CHECK: int* ok_mut_clone(int *a, int b) {
```

Comment:
I don't understand why the spacing changed here.

Reply:
Where we used to leave return values alone, we started changing them sometimes. This sounds unsatisfying as I type it, but it's the world we live in now. I could try to get to the bottom of it now, or later.